utils/download_utils.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+import numpy as np
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
```

Comment:
Shapely Point and Polygon imports didn't appear to be used below. Should they be removed?
```suggestion
```

Reply:
Yep, they should be removed!